Dynatrace/dynatrace-operator
# Undeploy operator gracefully

## Description

<no-ticket>

WDYT about this idea, doing dynakube CRs cleanup before deleting all operator components on `make undeploy`?

This would fix pods getting hanged-up on Terminating state forever because its mounted volume got deleted.

## How can this be tested?

Apply a dynakube and run `make undeploy`. You should see how the make command waits for the 'operator-managed' pods to get completely deleted before uninstalling the operator - no hanged-up/orphan pods
